assets/multiqc/multiqc_config.yaml

```diff
@@ -18,6 +18,8 @@ fn_clean_exts:
     - 'mLb'
     - '_peaks'
     - '_spp'
+    - '.spp.out'
```

Comment:
Good spot! I think `.spp` should do though because `MultiQC` will trim off the rest 👍

Reply:
Yep, so it does...but I cant figure out how to make that change here 🤔

Reply:
Just change it in your local repo and push a commit? Like you did with your last commit. I'm now wondering whether `*spp` could replace both of those lines...

Reply:
Or even just `spp` :laughing:

Reply:
I don’t think that this one is a regex / glob, so `*` probably won’t work. `spp` will, but it’ll leave the trailing `.` / `_`. Best to be verbose and just keep both :+1:

Reply:
There we go then 👍 `_spp` and `.spp` it is.